src/main/java/com/powsybl/openloadflow/AcLoadFlowFromCache.java

```diff
@@ -65,6 +65,22 @@ private void configureSwitches(LfTopoConfig topoConfig) {
                 LOGGER.warn("Actionable switch '{}' does not exist", switchId);
             }
         }
+        for (String transformerId : parametersExt.getActionableTransformersIds()) {
+            Branch<?> branch = network.getBranch(transformerId);
+            if (branch != null) {
+                topoConfig.addBranchIdWithRtcToRetain(transformerId);
+                topoConfig.addBranchIdWithPtcToRetain(transformerId);
+            } else {
+                ThreeWindingsTransformer tw3 = network.getThreeWindingsTransformer(transformerId);
```

Comment:
Of course, you know that here you can optimize a little bit more and just keep the leg where we have a tap changer.

Reply:
Does it worth it ?